functions/token.test.ts

```diff
@@ -19,12 +19,6 @@ const tokens = [
     image:
       'https://raw.githubusercontent.com/Uniswap/assets/master/blockchains/ethereum/assets/0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0/logo.png',
   },
-  {
```

Comment:
Should this change be in a separate PR?

Reply:
Yeah I already merged this change into `main` but I updated it on this branch to stop the failure and then just forgot to merge `main` back in. Should be good now!